windows_dev_setup.ps1

```diff
@@ -1,18 +1,29 @@
 # Windows Development Environment Bootstrap
-# Installs: Git, Python 3.10, CMake, VS 2022 Build Tools + Windows SDK, NVM, Node 18.20.4, Yarn 1.x
+# Installs: Git, Python, CMake, VS Build Tools + Windows SDK, NVM, Node, Yarn
 # Then clones/updates Vortex repository
 
 #Requires -RunAsAdministrator
 
+
 # Configuration
-$NODE_VERSION = "22.19.0"
+$NODE_VERSION = "22.19"
+$YarnVersion = "1.22.19"
```

Comment:
More of a discussion, but perhaps it's better to have a failfast approach here? If we're not able to extract the variables, fail instead of relying on in-script defaults

Reply:
Yea, see it now. Perhaps then we can first install git and clone the repo, then install Node? Not sure what @insomnious would think about this